packages/components/molecules/f-navigation-links/src/components/NavigationLinks.vue

```diff
@@ -54,25 +54,29 @@ export default {
 }
 
 .c-navigationLinks-item {
-    display: inline-block;
-    width: 100%;
+    display: block;
 }
+</style>
 
+<style lang="scss">
+/**
+* .c-navigationLinks-link is intentionally not scoped as the consuming router application (Nuxt.js / Vue Router) will add active classes directly to
+* the router-link within <f-link>. CoreWeb Nuxt custom active link classes are: `is-link-active` and `is-link-exactActive`.
+*/
 .c-navigationLinks-link {
-    display: block;
+    display: inline-block;
     padding: spacing() 0 spacing() spacing(x2);
     border-left: 2px solid $color-grey-30;
```

Comment:
Know this wasn't part of your PR, but could you update this while you're in there. These colours should point to alias tokens, not the global colour values.


```suggestion
    border-left: 2px solid $color-border-default;
```